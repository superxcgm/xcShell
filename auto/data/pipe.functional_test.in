touch hello
touch hello_world
touch world
ls | grep 'hello'
ls | grep 'hello' | grep 'world'