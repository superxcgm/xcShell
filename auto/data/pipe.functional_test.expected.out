xcShell_functional_test > touch hello
xcShell_functional_test > touch hello_world
xcShell_functional_test > touch world
xcShell_functional_test > ls | grep 'hello'
hello
hello_world
xcShell_functional_test > ls | grep 'hello' | grep 'world'
hello_world
xcShell_functional_test > alias ls='ls -G'
xcShell_functional_test > alias ll='ls -lh'
xcShell_functional_test > alias | grep ls
ll='ls -lh'
ls='ls -G'
xcShell_functional_test > alias | grep ls | grep ll
ll='ls -lh'
xcShell_functional_test > 